web/ui/react-app/src/App.css

```diff
@@ -212,6 +212,26 @@ div.time-input {
     margin-bottom: 20px;
 }
 
+.target-head {
+    font-weight: 700;
+    font-size: large;
+}
+
+.label-component {
+    margin-bottom: 1%;
+}
+
+.table-outer-layer {
```

Comment:
I don't see any change to the page at all if I remove these styles (and even cranking the numbers up here doesn't seem to change anything in the table layout?).

Reply:
oh. My earlier versions had some changes with this. But the newer code is independent of this style. Thanks for pointing.